sled-diagnostics/src/lib.rs

```diff
@@ -4,7 +4,8 @@
 
 //! Diagnostics for an Oxide sled that exposes common support commands.
 
-use futures::{StreamExt, stream::FuturesUnordered};
```

Comment:
Looks like the rebase picked up new code that uses `FuturesUnordered`: https://github.com/oxidecomputer/omicron/actions/runs/15056224877/job/42322633598?pr=8151#step:11:767